### ISSUE TITLE
LHCb tasks bug

### ISSUE THREAD

**Issue by rob-c** [ganga-devs/ganga#366]
I'm seeing the following on startup:

```
2016-04-08 17:01:20,890 MainThread Ganga.Utility                      :264 ERROR   : problems with loading Named Templates for GangaLHCb
2016-04-08 17:01:20,891 MainThread Ganga.Utility                      :265 ERROR   : Reason: name 'taskify' is not defined
```

Does anyone else know if this is serious or not?

Thanks,

Rob

**Comment by drmarkwslater** [comment]
That'll be due to the removal of a load of obsolete task stuff. I'll try to have a look at this ASAP.

**Comment by rob-c** [comment]
Thanks Mark!

**Comment by rob-c** [comment]
Are we able to fix this for 6.1.19?

**Comment by drmarkwslater** [comment]
Should be fixed with #377
